### PR TITLE
Added `currency` parameter to support ETH on Deribit

### LINKED ISSUE #24: ETH-PERPETUAL on Deribit returns BTC
```
================================
Message Received
Tue May 07 2019 15:58:26 GMT+0200 (GMT+02:00)
Message :
tvial(ETH-PERPETUAL) { balance() }
================================

Starting Deribit
{
  "kind": "future",
  "baseCurrency": "ETH",
  "currency": "USD",
  "minTradeSize": 1,
  "minTradeAmount": 1,
  "instrumentName": "ETH-PERPETUAL",
  "isActive": true,
  "settlement": "perpetual",
  "created": "2019-03-14 13:00:54 GMT",
  "tickSize": 0.01,
  "pricePrecision": 2,
  "expiration": "3000-01-01 08:00:00 GMT",
  "contractSize": 1
}

================================
Exchange : deribit
Symbol   : ETH-PERPETUAL
Session  : c81b0946-fda2-4724-91e0-eb9125a87021
Commands : balance()
================================

NOTIFY ACCOUNT BALANCE
Deribit: Equity: 0.1126 btc, available: 0.1126 btc, balance: 0.1126 btc, pnl: 0 btc.
Deribit exchange closing down
```

### PR DESCRIPTION
**Pull request by tomav**
Fixes #24

### CODE REVIEW

File: src/apis/deribit.js [hunk 1]
```diff
@@ -355,10 +355,11 @@ class DeribitApi extends ApiInterface {
 
     /**
      * Get account details
+     * @param currency
      * @returns {*}
      */
-    account() {
-        return this.makeAuthRequest('/api/v1/private/account', {});
+    account(currency) {
+        return this.makeAuthRequest('/api/v1/private/account', {currency: currency});
```
Comment (instabot42) [inline src/apis/deribit.js]: Please use the short form `{ currency }`

File: src/exchanges/exchange.js [hunk 2]
```diff
@@ -344,6 +344,24 @@ class Exchange {
         return { asset: 'btc', currency: 'usd' };
     }
 
+    /**
+     * Given a symbol (like ETH-PERPETUAL), figure out the pair (eth & usd)
+     * @param symbol
+     * @returns {*}
+     */
+    splitDashedSymbol(symbol) {
```
Comment (instabot42) [inline src/exchanges/exchange.js]: This does not correctly handle futures symbols and is Deribit specific, so should not be in the base exchange. This probably needs to move into balance_deribit.js and be simplified to just extract the symbol before the dash.

Comment (tomav) [reply src/exchanges/exchange.js]: Done.

File: src/apis/deribit.js (hunk 1 above)
Comment (tomav) [reply src/apis/deribit.js]: Done.

**Comment by tomav** [comment]
```
tomav(BTC-PERPETUAL) { balance() }
Working on it 👍
Deribit: Equity: 0.1126 BTC, available: 0.1126 BTC, balance: 0.1126 BTC, pnl: 0 BTC.

tomav(ETH-PERPETUAL) { balance() }
Working on it 👍
Deribit: Equity: 1.9645 ETH, available: 1.9645 ETH, balance: 1.9645 ETH, pnl: 0 ETH.
```

**Pull request merged**